lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1248,17 +1250,19 @@ emptyRandomWalletWithPasswd ctx rawPwd = do
             $ hex
             $ Byron.getKey
             $ Byron.generateKeyFromSeed seed pwd
-    pwdH <- liftIO $ T.decodeUtf8 . hex <$> encryptPasswordWithScrypt pwd
+    pwdH <- liftIO $ T.decodeUtf8 . hex @ByteString <$> scrypt pwd
     emptyByronWalletFromXPrvWith ctx "random" ("Random Wallet", key, pwdH)
   where
-    encryptPasswordWithScrypt =
-        fmap Scrypt.getEncryptedPass
-        . Scrypt.encryptPassIO Scrypt.defaultParams
-        . Scrypt.Pass
+    scrypt = encryptPasswordWithScrypt
         . CBOR.toStrictByteString
         . CBOR.encodeBytes
         . BA.convert
 
+encryptPasswordWithScrypt :: (BA.ByteArrayAccess a, BA.ByteArray b) => a -> IO b
```

Comment:
For posterity: The documentation of the `scrypt` package notes that the ByteString returned by `Scrypt.getEncryptedPass` contains not just the encrypted password, but also the parameters and salt used to encrypt it. In contrast, the `generate` function from the `cryptonite` package only seems to return that encrypted password.

The difference does not matter for the integration test, but it would matter for legacy wallets (hence the choice to keep the `scrypt` dependency in the core library.)

Reply:
Yes good point - thanks - this will need fixing.